poly-R.el

```diff
@@ -48,9 +48,36 @@
                :hostmode 'pm-host/R
                :innermodes '(pm-inner/fundamental))
   "Root polymode with R host intended to be inherited from."
-  :group 'polymodes
+  :group 'poly-R
   :type 'object)
 
+;;;###autoload
+(defun poly-r-set-markdown-flavour (&optional varname value)
+  "Flavour of markdown to use in Rmarkdown documents.
+\"GFM\" or \"gfm\" sets it to github-flavoured-markdown. Any
+other value uses standard markdown."
+  (set-default varname value)
+  (if (string-equal "gfm" (downcase value))
+      (add-to-list 'auto-mode-alist '("\\.[rR]md\\'" . poly-gfm+r-mode))
+    (add-to-list 'auto-mode-alist '("\\.[rR]md\\'" . poly-markdown+r-mode))))
+
+(defcustom poly-r-markdown-flavour "standard"
```

Comment:
I am hesitant on this change. It does not bring much compared to the standard `(add-to-list 'auto-mode-alist ...` . 

Extra defcustom seems to be a bit of an overkill indirection.  WDYT?